shell/platform/windows/keyboard_key_channel_handler.cc

```diff
@@ -32,6 +32,9 @@ static constexpr char kKeyDown[] = "keydown";
 // emitting a warning on the console about unhandled events.
 static constexpr int kMaxPendingEvents = 1000;
 
+// Chromium uses this bit in scancode to mark "extended" keys.
```

Comment:
Are you saying "Chromium" here because that's where you saw this pattern, or does this have something to do with Chromium on Windows running Flutter code? (I imagine it can only be the first, but the comment is unclear).